### PR TITLE
bump binaries revault 0.4

### PR DESCRIPTION
**Pull request by edouardparis**

### CODE REVIEW

File: aquarium.py [hunk 1]
```diff
@@ -32,11 +32,11 @@
 REVAULTD_SRC_DIR = os.path.join(SRC_DIR, "revaultd")
 REVAULT_GUI_SRC_DIR = os.path.join(SRC_DIR, "revault-gui")
 SHELL = os.getenv("SHELL", "bash")
-COORDINATORD_VERSION = os.getenv("COORDINATORD_VERSION", "master")
-COSIGNERD_VERSION = os.getenv("COSIGNERD_VERSION", "master")
-MIRADORD_VERSION = os.getenv("MIRADORD_VERSION", "master")
-REVAULTD_VERSION = os.getenv("REVAULTD_VERSION", "master")
-REVAULT_GUI_VERSION = os.getenv("REVAULT_GUI_VERSION", "master")
+COORDINATORD_VERSION = os.getenv("COORDINATORD_VERSION", "0.4.0")
+COSIGNERD_VERSION = os.getenv("COSIGNERD_VERSION", "0.4")
```
Comment (edouardparis) [inline aquarium.py:36]: note: cosignerd is tagged 0.4 instead of 0.4.0 in its repository

Comment (edouardparis) [reply aquarium.py:36]: same for gui

**Review by darosior — COMMENTED** [review]
I think the only good way to review this PR would be to merge #42 and #43, rebase it on top and test after deleting the `src/` repo. I didn't do this, but the tags look good to me. Indeed my mistake for the cosigner tag.

**Comment by edouardparis** [comment]
lets go
# v0.4.0: Babidi Somniosidae

**Pull request merged**